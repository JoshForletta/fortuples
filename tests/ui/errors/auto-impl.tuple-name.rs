#[fortuples::auto_impl]
#[tuples::tuple_name(A)]
trait Test {}

fn main() {}
