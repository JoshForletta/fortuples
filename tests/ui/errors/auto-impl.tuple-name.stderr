error: `auto_impl` won't use the `tuple_name`
 --> tests/ui/errors/auto-impl.tuple-name.rs:2:11
  |
2 | #[tuples::tuple_name(A)]
  |           ^^^^^^^^^^
